lib/screens/explore_screens/vouch/sponsor_tab/interactor/viewmodels/sponsor_bloc.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:seeds/datasource/remote/model/member_model.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+
+part 'sponsor_event.dart';
+
+part 'sponsor_state.dart';
+
+class SponsorBloc extends Bloc<SponsorEvent, SponsorState> {
+  SponsorBloc() : super(SponsorState.initial()) {
+    on<LoadUserSponsorList>(_loadUserSponsorList);
+  }
+
+  //To be finish on next pr
```

Comment:
This comment should be formatted as a TODO()

TODO(gguij004): Work to load list to be completed in next PR.